tests/driver_bmp180/main.c

```diff
@@ -73,12 +74,12 @@ int main(void)
         /* Get altitude in meters */
         int16_t altitude = bmp180_altitude(&dev, pressure_0);
 
-        printf("Temperature [°C]: %d.%d\n"
+        printf("Temperature [°C]: %i.%d\n"
```

Comment:
Both `%i` and `%d` have the same meaning, but they are wildly mixed in the rest of the file too, so whatever you like better 🤷‍♂

Reply:
So why change?